OsmAnd/src/net/osmand/plus/helpers/ImportHelper.java

```diff
@@ -1018,7 +1019,18 @@ private String saveImport(final GPXFile gpxFile, final String fileName, final bo
 					File file = new File(gpxFile.path);
 					if (!destinationExists) {
 						GPXDatabase.GpxDataItem item = new GPXDatabase.GpxDataItem(file, gpxFile.getColor(0));
+						item.setWidth(gpxFile.getWidth(null));
```

Comment:
Introduce new constructor **public GpxDataItem(File file, GPXFile gpx)** and drop all these setters.